ocpp/v16/enums.py

```diff
@@ -11,6 +13,13 @@ class StrEnum(str, Enum):  # pragma: no cover
 class Action(StrEnum):
     """An Action is a required part of a Call message."""
 
+    def __init__(self, *args, **kwargs):
+        warn(
+            "Action enum contains deprecated members and will be removed in "
+            "the next major release, please use snake case members."
+        )
```

Comment:
Can you include the category of warning to be a `DepricationWarning`?
See [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) and https://docs.python.org/3/library/warnings.html#warnings.warn

Reply:
Satisfied with commit - https://github.com/mobilityhouse/ocpp/pull/600/commits/9a405546bb160d698d0393225bfca561391467a8